app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -268,8 +268,9 @@ function handleCommentSubmit(event) {
             });
 
             var preview = document.getElementById("preview-div");
-            preview.classList.add("preview-toggle");
+            var togglePreview = document.querySelector('.preview-toggle');
```

Comment:
Where possible it would be good to use `const` rather than `var`.

Reply:
Does `js.erb` get transpiled to older js versions ?

Reply:
@Rafi993 we don't support IE11 and `const` doesn't need transpilation: https://caniuse.com/const

Reply:
I have now replaced `var` with `const`. I think there are few vars in the codebase that is used in few files they could also be replaced with `const` in a separate PR. I'll create issue for that